licensescout-core/src/main/java/org/aposin/licensescout/finder/FilesystemFinderHandler.java

```diff
@@ -73,7 +73,7 @@ public boolean isDirectory(final File file) {
      * {@inheritDoc}
      */
     @Override
-    public FileSystemEntryContainer createEntryContainer(final File ecBase) throws IOException {
+    public FileSystemEntryContainer createEntryContainer(final File ecBase, String filePath) throws IOException {
```

Comment:
Why the new variable is not used here?

Reply:
Both FileSystemFinderHandler and JarFinderHandler are derived from AbstractFinderHandler. The parameter was introduced to be able to give better error messages in JarFinderHandler. However, due to the class hierarchy, the signature of the method also changes in FileSystemFinderHandler, though the valu is not needed there (here, the path can be obtained from the File object)